### ISSUE TITLE
A11y_ SQLAzureDataStudioMAC_Create new server group_Keyboard: Only color has been used for the 'Group Color' section in the dialog 'Add server group' to convey information to the user.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19870]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#Keyboard;#AINotInScope;#WCAG1.4.1;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Tab till the connected server.
5. Tab to the 'New server group' button and hit return key.
6. 'Add server group' dialog appears.
7. Tab till the color palette in the dialog.

## Actual:
Only color has been used for the 'Group Color' section in the dialog 'Add server group' to convey information to the user.

## Expected:
Color should not be used as the only visual means of conveying information to the users. There should be some structure within the color that will help users with color blindness issues.

## User Impact:
Keyboard users will not know what each color stands for and users with color blindness will not see the color hence structure within the color to be added

## Attachment:
<img width="1440" alt="6_Only color" src="https://user-images.githubusercontent.com/98393966/176432175-4e5f5d04-143c-4559-9523-ea108e565ac2.png">

**Comment by sspatil123** [comment]
![image](https://user-images.githubusercontent.com/85863999/218422713-15e4c706-c461-45b2-bdd3-44055ca47738.png)
GithubTags:#Closed;Version 1.41.1
